unlock-app/src/services/web3Service.js

```diff
@@ -65,6 +65,17 @@ export default class Web3Service extends EventEmitter {
 
     this.web3 = new Web3(providers[provider])
 
+    if (providers[provider].enable) {
```

Comment:
I *think* this is the right approach BUT I want to make sure that the other pages (like `/` or `/about`) are still visible to users who do not have a Metamask wallet in their browser. Do you confirm that this works?

Also, do you mind writing a test for this? (probably mocking/stubbing `.enable()`?